### PR TITLE
[2354] Replace lingering references to cron.php with system:cron

### LINKED ISSUE #2354: update ubuntu installations example to use system cron
https://doc.owncloud.com/server/10.4/admin_manual/installation/ubuntu_18_04.html#set-up-a-cron-job

here is still the old command.

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #2354.

### CODE REVIEW

**Comment by settermjd** [comment]
@voroyam, please review this to ensure that all of the changes are valid?

File: modules/admin_manual/pages/installation/ubuntu_18_04.adoc [hunk 1]
```diff
@@ -160,7 +160,7 @@ occ config:system:set trusted_domains 1 --value="$myip"
 
 [source,console,subs="attributes+"]
 ----
-echo "*/15  *  *  *  * /usr/bin/php -f {install-directory}/cron.php" \
+echo "*/15  *  *  *  * /usr/bin/php occ system:cron" \
```
Comment (mmattel) [inline modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: Whenever there is a cron job to be entered, you need to ensure that the full path to the occ command is used.
Needs checking for all changes with regards to cron.
Beside the necessity of having the complete path to make it run successfully, imagine you have more than one instance on different domains on the same server.

Comment (settermjd) [reply modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: Thanks, @mmattel

Comment (settermjd) [reply modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: All references have been updated.

**Pull request merged**